src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -34,11 +34,6 @@
                 "title": "Sequentielle Testläufe",
                 "description": "Aktiviere diese Checkbox um zuerst Struktur- und erst im Anschluss Verhaltenstests duchzuführen. Dieses Feature hilft Studenten dabei, sich auf die aktuell anliegende Herausforderung zu konzentrieren. Bei Aktivierung werden im Test Repository zusätzliche Informationen bereitgestellt. ACHTUNG: Kann nicht nachträglich geändert werden!"
             },
-            "buildAndTestStudentSubmissionsAfterDueDate": {
```

Comment:
This label is still used in the programming-exercise detail view: 
![image](https://user-images.githubusercontent.com/33764166/66261841-d2f70d80-e7d4-11e9-80a9-6e490de3b138.png)

Reply:
Great that you found this!